src/review-form.ts

```diff
@@ -160,6 +197,28 @@ export class ReviewForm extends LitElement {
     `;
   }
 
+  private async setupRecaptcha(): Promise<void> {
+    this.recaptchaWidget = await this.recaptchaManager?.getRecaptchaWidget();
+  }
+
+  private async handleSubmit(e: Event): Promise<void> {
+    e.preventDefault();
+    if (!this.reviewForm.reportValidity()) return;
+
+    try {
+      const recaptchaToken = await this.recaptchaWidget?.execute();
```

Comment:
I would probably do a check above this to make sure the `recaptchaWidget` is available and prevent submission if it's not. Then `recaptchaWidget` won't be optional and `this.recaptchaToken = recaptchaToken ?? '';` will never fall back to an empty string.